### DISCUSSION TITLE
Getting stuck at setting up website locally

### DISCUSSION THREAD

**Discussion by SnehalSrivastava27** [ethereum/ethereum-org-website#13178] [Q&A]
Here is the screenshot attached please help I tried three times but still there is an issue
![IMG-20240615-WA0063](https://github.com/ethereum/ethereum-org-website/assets/118104081/e20bc9e4-3e8d-4ef7-9aa6-794985c92311)

**Comment by corwintines** [comment]
This is a known hydration issue. Basically what is happening is what is on the server doesnt match what gets rendered on the client @SnehalSrivastava27. You can just close this out and continue developing, while the experience is a bit annoying, it isn't a blocker and seems like you have the project running :)
